src/depfile.rs

```diff
@@ -72,19 +72,31 @@ pub fn parse<'a>(scanner: &mut Scanner<'a>) -> ParseResult<Deps<'a>> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    #[test]
-    fn test_parse() {
+
+    fn must_parse<'a>(s: &'a str) -> Deps<'a> {
         let mut scanner =
-            Scanner::new("build/browse.o: src/browse.cc src/browse.h build/browse_py.h\n\0");
-        let deps = match parse(&mut scanner) {
+            Scanner::new(s);
```

Comment:
Can you run 'cargo fmt'

Reply:
```
error: no such subcommand: `fmt`

	Did you mean `fix`?
```

The fix was to `apt-get install rustfmt`, but that wasn't clear to me, and surprisingly hard to google for.

Done.